### PR TITLE
Allow adding custom CSS

### PR DESCRIPTION
**Pull request by amitaibu**
#158

### CODE REVIEW

**Comment by ceoaliongroo** [comment]
test ok web

### example:

apply:
http://localhost:9000/#/custom-css/koc
reset:
http://localhost:9000/#/custom-css/reset

@amitaibu, @mushon  to modify custom css:
Change in config.json property CUSTOM_CSS

File: app/scripts/controllers/main.js [hunk 1]
```diff
@@ -1,8 +1,30 @@
...
 
 angular.module('publicEducationApp')
-  .controller('ListMarkersCtrl', function ($scope, $window, Leaflet, storage, Marker, Geolocation, $location, $timeout, IS_MOBILE) {
+  .controller('ListMarkersCtrl', function ($scope, $window, Leaflet, storage, Marker, Geolocation, $location, $timeout, IS_MOBILE, $routeParams, CUSTOM_CSS) {
 
+    // Storage custom css information in the LocalStorage.
+    storage.bind($scope,'isCustomCss', {defaultValue: false});
+    storage.bind($scope,'customCss', {defaultValue: 'reset-custom-css'});
+
+    if ($routeParams.route === CUSTOM_CSS.route) {
```
Comment (amitaibu) [inline app/scripts/controllers/main.js]: Maybe lets simplify and have `if ($routeParams.route === 'apply') {`

File: app/scripts/controllers/main.js [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
+      // Activate custom css.
+      $scope.isCustomCss = true;
+      $scope.customCss = $scope.customCss + ' ' +  CUSTOM_CSS.css;
```
Comment (amitaibu) [inline app/scripts/controllers/main.js]: `$scope.customCss =  CUSTOM_CSS`

File: config.example.json [hunk 3]
```diff
@@ -31,6 +35,10 @@
     "URL_STORE": {
       "iOS": "https://itunes.apple.com/app/chrome/id535886823?mt=8",
       "android" : "https://play.google.com/store/apps/details?id=com.android.chrome"
+    },
+    "CUSTOM_CSS" : {
```
Comment (amitaibu) [inline config.example.json]: "CUSTOM_CSS" : "koc"

File: app/styles/main.scss [hunk 4]
```diff
@@ -43,6 +43,29 @@ body{
...
 }
 
+// Reset CSS to prepare css.
+.reset-custom-css {
+  margin: 0;
+  width: 100%;
+  height: 100%;
+}
+
+// Example custom css.
```
Comment (amitaibu) [inline app/styles/main.scss]: Remove example.

**Comment by ceoaliongroo** [comment]
Add custom css in config.json, property CUSTOM_CSS.

to apply: http://localhost:9000/#/custom-css/apply
to reset: http://localhost:9000/#/custom-css/reset

**Comment by mushon** [comment]
I would need some help merging this. After merge I can't even see the map.

## 

Mushon Zer-Aviv
Mushon.com (http://Mushon.com) | Shual.com (http://Shual.com) | @Mushon (http://twitter.com/mushon)

**Comment by mushon** [comment]
If it works for you please update it in gh-pages so I can test it on the exhibition's screen.

**Comment by amitaibu** [comment]
> I would need some help merging this.

merge where?

> If it works for you please update it in gh-pages 

Only on my computer I see some weird black rectangle on initial reload.

**Comment by mushon** [comment]
I mean merge locally.
It doesn't show the map or the add-marker. 

Mushon Zer-Aviv
Mushon.com | Shual.com | @Mushon

**Comment by amitaibu** [comment]
Are you trying to merge to into master?

**Comment by amitaibu** [comment]
@mushon I'd be happy if you could try to check it, as the issue is a CSS one. Once wrapping those pages with our div, the angular-leaflet-map is no longer `height: 100%`

**Comment by mushon** [comment]
@amitaibu please pull this now and update gh-pages. It's looking good

**Pull request merged**